helm/prometheus-meta-operator/templates/ingress-oauth.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: {{ include "resource.default.name" . }}
-  namespace: "monitoring"
+  namespace: {{ include "resource.default.namespace" . }}
```

Comment:
I was not so sure about this, because it does not matter where pmo is installed. This ingress always has to be in `monitoring` namespace.

Reply:
Do you want me to change it back?

Reply:
I changed it back.